engineapi/engineapi/models.py

```diff
@@ -315,6 +315,7 @@ class CallRequest(Base):
     parameters = Column(JSONB, nullable=False)
 
     expires_at = Column(DateTime(timezone=True), nullable=True, index=True)
+    live_at = Column(DateTime(timezone=True), server_default=utcnow(), nullable=False)
```

Comment:
I think it should be `nullable=true`. If `live_at = NULL`, then we always display that `call_request`. What do you think?

Reply:
It will require to add additional logic which I would prefer to not do if the result will be same - just return list of call_requests